### DISCUSSION TITLE
deducting fees during money transfers.

### DISCUSSION THREAD

**Discussion by mzainmushtaq33** [bavix/laravel-wallet#709] [Q&A]
i there any way to deduct fee during money transfer ??

**Comment by rez1dent3** [comment]
Hello. Here are the interfaces for taxes, but they were planned to be used for goods. 
https://github.com/bavix/laravel-wallet/blob/0be5309988d05d774e4d541ddc1e4dc8b518ee1d/src/Interfaces/MaximalTaxable.php#L9
https://github.com/bavix/laravel-wallet/blob/0be5309988d05d774e4d541ddc1e4dc8b518ee1d/src/Interfaces/MinimalTaxable.php#L9
https://github.com/bavix/laravel-wallet/blob/0be5309988d05d774e4d541ddc1e4dc8b518ee1d/src/Interfaces/Taxable.php#L9-L14

If the "$to" object will inherit the interface, then the commission will be automatically charged.

https://github.com/bavix/laravel-wallet/blob/b4e8ea1b06ba407a6450f025315aa44f59114391/src/Services/PrepareService.php#L92

**Reply by rez1dent3** [reply]
@mzainmushtaq33 Have questions?
